FHEM/14_SD_BELL.pm

```diff
@@ -202,62 +202,93 @@ sub Parse($$) {
 	my $deviceCode = $rawData;
 	my $devicedef;
 
-	Log3 $iohash, 4, "$ioname: SD_BELL_Parse protocol $protocol $hash_name doubleCode=".$models{$hash_name}{doubleCode}.", bitData $bitData";	
+	Log3 $iohash, 4, "$ioname: SD_BELL_Parse protocol $protocol $hash_name doubleCode=".$models{$hash_name}{doubleCode}." rawData=$rawData";	
 
 	## loop to view SD_BELL defined defptr ##
 	if ($protocol == 41) {
 		foreach my $d(sort keys %{$modules{SD_BELL}{defptr}}) {
-			Log3 $iohash, 5, "$ioname: SD_BELL_Parse Check defptr - $d" if ($d =~ /$protocol/s);
-			Log3 $iohash, 4, "$ioname: SD_BELL_Parse Check defptr - $rawData is already registered" if ($d =~ /$rawData/s);
-			my @doubleCode = split(" ",$d) if ($d =~ /$rawData/s);																															# split two RAWMSG from protocol in def
-			$doubleCode_known = $doubleCode[1] if ($d =~ /$rawData/s);																													# RAWMSG are in split RAWMSG
-			Log3 $iohash, 5, "$ioname: SD_BELL_Parse Check defptr - $doubleCode_known found";
+			Log3 $iohash, 4, "$ioname: SD_BELL_Parse Check defptr - $d is defined!" if ($d =~ /$protocol/s);
+			if ($d =~ /$rawData/s) {
+				my @doubleCode = split(" ",$d);								# split two RAWMSG from protocol in def 41 BA7983D3_3286D393
```

Comment:
Ich frage mich, wann und wieso zwei rawmsgs in die def eines Gerätes kommen.
Kannst Du mich hier aufklären?

Reply:
Das Gerät sendet 15x den einen Code und sofort danach 15x den 2. Code. Das haben wir ausgelesen direkt.

Der Code reagiert auf den ersten Code, legt diesen ab  und wartet auf den 2. der dazugehört. Durch einen Vergleich ob es die selbige Nachricht ist, so wird gefiltert. Wenn diese unterscchiedlich ist wird dann im "Marker" der 2. Code drangeknüpft weil wenn man nur einen Code in FHEM registrieren würde, so würde ich für 1 klingel 2 Geräte erhalten was nicht stimmt. Es gibt ja auch nur einen Klingelknopf.

Sinngemäß, 
- erster Code mit dem Vermerk das noch was fehlt https://github.com/RFD-FHEM/RFFHEM/blob/6a61f8660c5aeb580d2c3c3b2608e4a42dc8f17a/FHEM/14_SD_BELL.pm#L235
- beim 2. Durchlauf der Split um _doublecode zu ersetzen mit dem anderen Code
- ins Define dann 3286D393_BA7983D3 schreiben um beim Empfang vergleichen zu können welcher Code eintrifft.

Reply:
Sind das zwei getrennte Funktelegramme oder wie muss ich mir das vorstellen?